tests/solidity/test_withdraw_shares.py

```diff
@@ -0,0 +1,95 @@
+"""Withdraw shares tests that match those being executed in the solidity repo."""
+import unittest
+
+import numpy
```

Comment:
The convention is to `import numpy as np`, per [here](https://numpy.org/doc/stable/user/absolute_beginners.html#how-to-import-numpy)